### ISSUE TITLE
Pacific/Bougainville timezone missing from the default package

### ISSUE THREAD

**Issue by Max-Might** [srawlins/timezone#59]
The `Pacific/Bougainville` timezone is missing from the **default** package but is present in the **all** package.

I think it has not been deprecated [according to this list](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).

**Comment by DominicOrga** [comment]
I was able to use deprecated locations by importing `'package:timezone/data/latest_all.dart'` instead of `'package:timezone/data/latest.dart'`.

**Comment by Max-Might** [comment]
@DominicOrga yeah, that is what I said. The thing is that this timezone is no deprecated so it should be in the latest package too.
